floris/simulation/farm.py

```diff
@@ -180,11 +200,36 @@ def expand_farm_properties(
             sorted_coord_indices,
             axis=2
         )
+        self.ref_density_cp_cts_sorted = np.take_along_axis(
```

Comment:
Need a similar method for fTilts?

Reply:
Using the pattern from Ct/power interpolants, an array of just the different interpolators is kept (one for each kind of turbine) and the turbine type is checked to know which interpolant (fTilt) to use, so don't need to sort them.